### PR TITLE
Update tidb-monitoring-api.md

### PR DESCRIPTION
**Pull request by xiaoyangwang5**
### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/docs-cn/pull/3196

### CODE REVIEW

File: tidb-monitoring-api.md [hunk 1]
```diff
@@ -0,0 +1,364 @@
+---
+title: TiDB 集群监控 API
+category: how-to
+aliases: ['/docs-cn/stable/how-to/monitor/tidb-monitoring-api/']
```
Comment (TomShawn) [inline tidb-monitoring-api.md:4]: @xiaoyangwang5 请在 TOC 目录中添加本文档的链接。
```suggestion
```

**Comment by TomShawn** [comment]
@xiaoyangwang5 请签署一下 CLA，这样 PR 才能合。

**Comment by TomShawn** [comment]
@CocaLi PTAL

**Comment by TomShawn** [comment]
@xiaoyangwang5 PR 中的内容是已有官网文档内容：[TiDB 集群监控](https://pingcap.com/docs-cn/stable/monitor-a-tidb-cluster/)。

**Comment by TomShawn** [comment]
@xiaoyangwang5 Will you add another document `[监控部署](/monitor-deploy.md)`? If not, this PR cannot be merged because you have referenced `监控部署` in this PR.

**Comment by xiaoyangwang5** [comment]
@TomShawn I add this document [监控部署](/monitor-deploy.md)

**Comment by TomShawn** [comment]
This PR must be merged after https://github.com/pingcap/docs-cn/pull/3196.

File: tidb-monitoring-api.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
+---
+title: TiDB 集群监控 API
+category: how-to
+aliases: ['/docs-cn/stable/how-to/monitor/monitor-a-cluster/']
```
Comment (TomShawn) [inline tidb-monitoring-api.md:4]:
```suggestion
```

File: tidb-monitoring-api.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+```
+
+## 使用 metrics 接口
+
+Metrics 接口用于监控整个集群的状态和性能。
+
+- 如果使用 TiDB Ansible 部署 TiDB 集群，监控系统（Prometheus 和 Grafana）会同时部署。
+- 如果使用其他方式部署 TiDB 集群，在使用 metrics 接口前，需先[部署 Prometheus 和 Grafana](deploy-monitoring-services.md)。
+
+成功部署 Prometheus 和 Grafana 之后，[配置 Grafana](deploy-monitoring-services.md)。
```
Comment (TomShawn) [inline tidb-monitoring-api.md:98]:
```suggestion
成功部署 Prometheus 和 Grafana 之后，[配置 Grafana](/deploy-monitoring-services.md)。

```

File: tidb-monitoring-api.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+    }
+  ]
+```
+
+## 使用 metrics 接口
+
+Metrics 接口用于监控整个集群的状态和性能。
+
+- 如果使用 TiDB Ansible 部署 TiDB 集群，监控系统（Prometheus 和 Grafana）会同时部署。
+- 如果使用其他方式部署 TiDB 集群，在使用 metrics 接口前，需先[部署 Prometheus 和 Grafana](deploy-monitoring-services.md)。
```
Comment (TomShawn) [inline tidb-monitoring-api.md:96]:
```suggestion
- 如果使用其他方式部署 TiDB 集群，在使用 metrics 接口前，需先[部署 Prometheus 和 Grafana](/deploy-monitoring-services.md)。

```

**Comment by TomShawn** [comment]
The change of this PR is migrated to https://github.com/pingcap/docs-cn/pull/3196.

**Comment by zz-jason** [comment]
> The change of this PR is migrated to #3196.

Do we need to review this PR?

**Comment by TomShawn** [comment]
> > The change of this PR is migrated to #3196.
> 
> Do we need to review this PR?

No. Please review #3196, thanks!

**Pull request closed without merging**